core/mondoo-windows-workstation-security.mql.yaml

```diff
@@ -0,0 +1,182 @@
+        Microsoft designed bitLocker to provide encryption for disk volumes. It is a free and built-in feature in many Windows versions, including Windows Vista and Windows 10. BitLocker asks users for a password, generates a recovery key, and encrypts the entire hard drive.
+      remediation: |
+        Turn on device encryption
+
+        1. Sign in to Windows with an administrator account (you may have to sign out and back in to switch accounts). For more info, see Create a local or administrator account in Windows.
+
+        2. Select the Start  button, then select Settings  > Update & Security > Device encryption. If Device encryption doesn't appear, it isn't available. You may be able to use standard BitLocker encryption instead. Open Device encryption in Settings.
+
+        3. If device encryption is turned off, select Turn on.
+
+        Turn on standard BitLocker encryption
+
+        1. Sign in to your Windows device with an administrator account (you may have to sign out and back in to switch accounts). For more info, see Create a local or administrator account in Windows.
+
+        2. In the search box on the taskbar, type Manage BitLocker and then select it from the list of results. Or, select the Start  button, and then under Windows System, select Control Panel. In Control Panel, select System and Security, and then under BitLocker Drive Encryption, select Manage BitLocker.
+
+        Note: You'll only see this option if BitLocker is available for your device. It isn't available on Windows 10 Home edition.
+
+        3. Select Turn on BitLocker and then follow the instructions.
+    refs:
+      - title: Turn on device encryption
+        url: https://support.microsoft.com/en-us/windows/turn-on-device-encryption-0c453637-bc88-5f74-5105-741561aae838#:~:text=Turn%20on%20standard%20BitLocker%20encryption,-Sign%20in%20to&text=In%20the%20search%20box%20on,is%20available%20for%20your%20device.
+    query: windows.bitlocker.volumes.all( protectionStatus["text"] == "Protected" )
+  - uid: mondoo-windows-workstation-security-secure-boot-is-enabled
+    title: Ensure Secure Boot is enabled
+    severity: 100
+    docs:
+      desc: |
+        Secure Boot is a boot integrity feature that is part of the Unified Extensible Firmware Interface (UEFI) industry standard. Most modern computer systems are delivered to customers with a standard Secure Boot policy installed.
+      remediation: |
+        Enable Secure Boot in your computer's firmware.
+    refs:
+      - title: UEFI Secure Boot Customization
+        url: https://media.defense.gov/2020/Sep/15/2002497594/-1/-1/0/CTR-UEFI-Secure-Boot-Customization-UOO168873-20.PDF
+    query: |
+      command("Confirm-SecureBootUEFI").stdout.trim.downcase == 'true'
```

Comment:
Since this requires powershell, we should probably use `powershell` resource

Reply:
@chris-rock thanks for the hint, fixed